content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md

```diff
@@ -0,0 +1,256 @@
+---
+title: influxctl release notes
+list_title: influxctl
+description: >
+  Release notes for the `influxctl` CLI used to manage InfluxDB v3 clusters.
+menu:
+  influxdb_cloud_dedicated:
+    identifier: influxctl-release-notes
+    name: influxctl
+    parent: Release notes
+weight: 202
+---
+
+## v2.2.0 {date="2023-10-27"}
+
+### Features
+
+- Allow token authentication for InfluxDB Clustered.
+
+---
+
+## v2.1.0 {date="2023-10-26"}
+
+### Features
+
+- Add `--format` flag to [`influxctl token create`](/influxdb/cloud-dedicated/reference/cli/influxctl/token/create/)
+  to specify the command output format.
+
+### Bug fixes
+
+- Use correct account and cluster names when specified in command flags.
+- Make `influxctl database list` return an empty array instead of _null_ when
+  there are no databases to list.
+
+### Dependency updates
+
+- Update to Go 1.21.3.
+- Update `github.com/jedib0t/go-pretty/v6` from 6.4.7 to 6.4.8.
+- Update `go.uber.org/zap` from 1.25.0 to 1.26.0.
+- Update `golang.org/x/mod` from 0.12.0 to 0.13.0.
+- Update `golang.org/x/net` from 0.15.0 to 0.17.0.
+- Update `golang.org/x/oauth2` from 0.12.0 to 0.13.0.
+- Update `google.golang.org/grpc` from 1.58.0 to 1.59.0.
+
+### Miscellaneous
+
+- Automate Homebrew tap releases for `influxctl`.
+
+---
+
+## v2.0.4 {date="2023-09-14"}
+
+### Bug fixes
+
+- Validate Microsoft Entra ID (formerly Azure Active Directory) token and device URLs.
+- Only validate Microsoft Entra ID configuration when getting a token.
+
+### Dependency updates
+
+- Update to Go 1.21.
+
+---
+
+## v2.0.3 {date="2023-09-12"}
+
+### Bug fixes
+
+- Add pagination support to [`influxctl token list`](/influxdb/cloud-dedicated/reference/cli/influxctl/token/list/).
```

Comment:
This added pagination to user list, account list, and token list.

Reply:
Yeah, I left `user` and `account` out because I thought they were hidden. I see now that `user list` is in the CLI help. The `account` command is hidden.